{
    "namespace": "django.contrib.gis.geos.error",
    "docstring": "",
    "classes": [
        {
            "docstring": "\"The base GEOS exception, indicates a GEOS-related error.\"",
            "name": "GEOSException",
            "namespace": "django.contrib.gis.geos.error.GEOSException",
            "methods": []
        }
    ],
    "functions": [],
    "package_name": "django.contrib.gis.geos"
}